src/helpers/registrationValidation.js

```diff
@@ -0,0 +1,30 @@
+const { check } = require('express-validator');
+
+const registrationValidators = {
+  namesValidator: (name) =>
+    check(name)
+      .trim()
+      .notEmpty()
+      .withMessage(`${name} should not be empty`)
+      .isLength({ min: 4 })
+      .withMessage(`${name} must be at least 4 characters long`)
```

Comment:
I made it two characters later, to make more sense, because some people might have a name with two chars. But of course good point and three is make more sense than 4